frontend/src/components/SupportedAppTitle.tsx

```diff
@@ -10,18 +10,34 @@ type SupportedAppTitleProps = {
 };
 
 const SupportedAppTitle: React.FC<SupportedAppTitleProps> = ({ odhApp, showProvider = false }) => {
+  const { title, icon } = React.useMemo(() => {
+    const split = odhApp.spec.displayName?.split(' ');
```

Comment:
`displayName` is not optional in our type and the CRD even specifies it as required -- avoid unnecessary conditionals.